src/test/Components/CreateImageWizard/steps/Services/Services.test.tsx

```diff
@@ -360,5 +360,3 @@ describe('Services edit mode', () => {
     expect(receivedRequest).toEqual(expectedRequest);
   });
 });
-
-// TO DO 'Services step' -> 'revisit step button on Review works'
```

Comment:
Sneaked this in, the test was already added and I didn't feel like this requires a separate PR :shrug: